atm.js

```diff
@@ -1 +1,74 @@
-// your code goes here!
+var checkingsBalance = 0
+var savingsBalance = 0
+var checkingDepositButton = document.querySelector("#checkingDepositButton")
+var checkingWithdrawalButton = document.querySelector("#checkingWithdrawalButton")
+var savingsDepositButton = document.querySelector("#savingsDepositButton")
+var savingsWithdrawalButton = document.querySelector("#savingsWithdrawalButton")
+var bothBalances = function () {
```

Comment:
I like that you turned this into a function! :+1: This helps to cut down on repeating yourself. A new line above this function might help make it more readable though.